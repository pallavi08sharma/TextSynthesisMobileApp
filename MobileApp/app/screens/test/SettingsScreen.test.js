import React from 'react';
import renderer from 'react-test-renderer';
import SettingsScreen from "../SettingsScreen"

describe('<SettingsScreen />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<SettingsScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});