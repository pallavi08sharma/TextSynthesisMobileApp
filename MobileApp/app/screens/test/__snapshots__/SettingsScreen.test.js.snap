// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SettingsScreen /> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 16,
        "fontWeight": "700",
      }
    }
  >
    Settings Screen
  </Text>
</View>
`;
